htdocs/website/websiteaccount_card.php

```diff
@@ -249,6 +249,7 @@
 	print '<input type="hidden" name="action" value="update">';
 	print '<input type="hidden" name="backtopage" value="'.$backtopage.'">';
 	print '<input type="hidden" name="id" value="'.$object->id.'">';
+	print '<input type="hidden" name="token" value="'.newToken().'">';
```

Comment:
Can you remove this line, it is duplicated.

Reply:
You're right, I removed this duplicated line